src/org/parosproxy/paros/view/FindDialog.java

```diff
@@ -108,6 +124,48 @@ private void initialize() {
         txtFind.requestFocus();
         this.getRootPane().setDefaultButton(btnFind);
         pack();
+        this.setVisible(true);
+	}
+	
+	private void registerParent(Object parent) {

```

Comment:
This could be inlined, it's just used in one place (also, it could just `put` without needing to check if it's not contained, we already know it isn't).